elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -263,4 +263,12 @@ default boolean supportsSorting(Class<?> entityClass, Sorting sorting) {
     default boolean supportsPagination(Class<?> entityClass, FilterExpression expression) {
         return true;
     }
+
+    /**
+     * @return UUID of transaction
+     *
+     */
+    default UUID getId() {
```

Comment:
Given that this is an interface, I'm not sure a default implementation makes sense here. You can leave the method, but remove default and the implementation.